.gitignore

```diff
@@ -20,7 +20,7 @@ config.nice
 config.status
 config.sub
 configure
-configure.in
+configure.ac
```

Comment:
This is a list of files that should not be tracked in version control. Keep the deleted line.

While configure.ac should indeed not be tracked, configure.in should also not be tracked.

Reply:
I see. I've missed PHP <= 7.1 which still have configure.in. :+1: Ok.